package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ppa-django",
+  "description": "Django web application for the Princeton Prosody Archive",
+  "repository": "https://github.com/Princeton-CDH/ppa-django.git",
+  "author": "The Center for Digital Humanities at Princeton <cdh@princeton.edu>",
```

Comment:
cdh@princeton isn't actually one of our email addresses - you could use cdh-info, but may cdhdevteam might make more sense here